frontend/src/features/projects/common/forms/DocumentationForm.tsx

```diff
@@ -18,13 +17,7 @@ const DocumentationForm = ({ isReadOnly, tasks, showNote = false }: IDocumentati
     <Fragment>
       <h3>Documentation</h3>
       <TasksForm tasks={tasks ?? []} isReadOnly={isReadOnly} />
-      {showNote && (
-        <ProjectNotes
-          label="Note"
-          field={`notes[${NoteTypes.Documentation}].note`}
```

Comment:
Using array positioning is an invalid implementation and will result in data corruption